packages/react-dom/src/client/ReactInputSelection.js

```diff
@@ -56,6 +56,11 @@ function isInDocument(node) {
   );
 }
 
+/**
+ * @getActiveElementDeep: Accessing the contentDocument of a HTMLIframeElement
```

Comment:
Sorry to be nickpicking -- could you make this a simple inline comment please? We're not using jsdoc in new code anymore and we'll probably delete the more useless ones